packages/manager/modules/bm-server-components/src/general-information/installation/ovh/server-installation-ovh.controller.js

```diff
@@ -2207,8 +2207,8 @@ export default class ServerInstallationOvhCtrl {
       hardwareRaid: [],
       partitioning: {
         disks:
-          this.$scope.informations.nbDisk > 2 &&
-          this.$scope.installation.nbDiskUse > 1
+          this.$scope.informations.nbDisk > 1 &&
+          this.$scope.installation.nbDiskUse
```

Comment:
Why delete `> 1` on the second line ?

Reply:
`>= 1` may be more readable ? As you want

Reply:
because if server has only 1 single disk, then the dropdown to choose the number of disks is not displayed since user can only choose 1 disk. Therefore we don't rely on nbDiskUse and systematically set the value to 0 meaning all disks in the API